src/identity/contactMapping.ts

```diff
@@ -60,6 +64,9 @@ import { call, delay, put, race, select, spawn, take } from 'typed-redux-saga'
 const TAG = 'identity/contactMapping'
 export const IMPORT_CONTACTS_TIMEOUT = 1 * 60 * 1000 // 1 minute
 
+const CONTACTS_PERMISSION =
+  Platform.OS === 'ios' ? PERMISSIONS.IOS.CONTACTS : PERMISSIONS.ANDROID.READ_CONTACTS
```

Comment:
This seems duplicated in many files, can we move this to a shared constant?
Given the number of places where we just check permissions === GRANTED, might be worth having a wrapper that returns a boolean, although I believe we can't use that directly in the Send recipient screen as it needs to access the other statuses.